openmctStaticServer/domainObjTypes.js

```diff
@@ -0,0 +1,32 @@
+const DOMAIN_OBJECTS_TYPES = {
+    "yarpopenmct.sensormeas": {
+        name: 'iCub Sensor Telemetry Entry',
+        description: 'Sensor measurements from one or multiple iCub sensors published on a single port.',
+        icon: 'icon-telemetry',
+        reportSchedule: [ReportSchedule.Realtime,ReportSchedule.Historical]
+    },
+    "yarpopenmct.camimage": {
+        name: 'iCub Eye Camera Telemetry Entry',
+        description: 'Camera images from one of the iCub eye cameras published on a single port.',
+        icon: 'icon-camera',
+        reportSchedule: [ReportSchedule.Realtime,ReportSchedule.Historical]
+    },
```

Comment:
I remember from last meeting that using `ReportSchedule.Historical` for camera images might be a issue. Isn't it?

Reply:
Yes, my intention was to do that change in a new PR. But actually I just realized that the plotter should be able to handle a buffer of samples and does so, even if we don't request historical data, as it buffers the samples received through the (realtime) subscription.
On top of that, I realized that the multiple lines appear even in a single sample (refer to https://github.com/ami-iit/yarp-openmct/issues/71#issuecomment-1063569388).
For those reasons, I suspect that the multiple lines flickering is due to a whole other reason, and using `ReportSchedule.Historical` shouldn't be a problem, although it could slow down<sup>*</sup> the display when clicking on the "Camera View Left/Right Eye" entry.

(*) Actually I haven't seen any noticeable difference when using or not the "ReportSchedule.Historical" option.

Reply:
FYI, every received sample is pushed o a history buffer, as per the code in the "imagery" openmct plugin `<openmct>/src/plugins/imagery/mixins/imageryData.js`:
```js
if (parsedTimestamp >= bounds.start && parsedTimestamp <= bounds.end) {
    let image = this.normalizeDatum(datum);
    if (image) {
        this.imageHistory.push(image);
    }
}
```